client/blocks/reader-full-post/action-links.jsx

```diff
@@ -7,7 +7,8 @@ import React from 'react';
  * Internal dependencies
  */
 import CommentButton from 'blocks/comment-button';
-import LikeButton from 'reader/like-button';
+//import LikeButton from 'reader/like-button'; // TODO why reader/like-button vs. blocks/like-button ?
```

Comment:
`reader/like-button` is a simple wrapper around `blocks/like-button` that fires our like/unlike stats (e.g. `calypso_reader_article_liked`).